AndelaOSP/art-dashboard
# #159958736 Move add asset from modal to page

## What does this PR do?

Moves the add asset functionality from a modal to a page.

## Description of Task to be completed?

### Acceptance Criteria

```gherkin
Scenario: Adding an asset should be done from a page
Given A logged in user
When I click on the "Add Asset" button
Then I should see a page that allows me to add asset details
```

**Notes:**
This is currently implemented as a modal. It should be moved to its own page.

## How should this be manually tested?

* Log in
* Click on the `Add Asset` button on the navigation bar
* You should be redirected to the `/assets/create` page where you can add an asset

## What are the relevant pivotal tracker stories?

[159958736](https://www.pivotaltracker.com/story/show/159958736)

## Any background context you want to add?

N/A

## Important notes

N/A

## Packages installed

N/A

## Deployment note

N/A

## Related PRs branch | PR (branch_name) | (pr_link)

N/A

## Todos

## Screenshots

* When form is empty
<img width="1677" alt="screen shot 2018-10-04 at 22 58 44" src="https://user-images.githubusercontent.com/31339738/46499642-4c0dfe80-c829-11e8-8e32-82c3d3552dc8.png">

* When device details are filled correctly
<img width="1677" alt="screen shot 2018-10-04 at 22 59 28" src="https://user-images.githubusercontent.com/31339738/46499682-5e883800-c829-11e8-880a-5271ef8435e4.png">

* Device specifications section
<img width="1677" alt="screen shot 2018-10-04 at 22 59 56" src="https://user-images.githubusercontent.com/31339738/46499708-7495f880-c829-11e8-9296-32c3c2eec0f3.png">
